weave/ops_primitives/file.py

```diff
@@ -55,6 +55,10 @@ def _file_dict_from_manifest(
     if "ref" in base_dict:
         base_dict["url"] = base_dict["ref"]
     else:
+        print(art.uri_obj, flush=True)
+        print(art.uri_obj.__class__, flush=True)
+        uri_obj_with_path = art.uri_obj.with_path(file.path)
```

Comment:
this variable seems to be unused? Leftover from a prior idea?

Reply:
oops thank you